tests/load/duckdb/test_duckdb_client.py

```diff
@@ -147,12 +148,12 @@ def test_keeps_initial_db_path() -> None:
         assert p.get_local_state_val("duckdb_database").lower() == os.path.abspath(db_path).lower()
         # new pipeline context took over
         # TODO: restore pipeline context on each call
-        assert conn.credentials.database.lower() != os.path.abspath(db_path).lower()
+        assert conn.credentials._conn_str().lower() != os.path.abspath(db_path).lower()
 
 
 def test_duckdb_database_delete() -> None:
     db_path = "_storage/path_test_quack.duckdb"
-    p = dlt.pipeline(pipeline_name="quack_pipeline", credentials=db_path, destination="duckdb")
+    p = dlt.pipeline(pipeline_name="quack_pipeline", destination=duckdb(credentials=DuckDbCredentials(db_path)))
```

Comment:
this should still work. we just need to create the instance ourselves if stirng is passed. I explained that above